daffodil-tdml-processor/src/test/scala/org/apache/daffodil/processor/tdml/TestDaffodilC.scala

```diff
@@ -218,7 +218,6 @@ class TestDaffodilC {
     cr match {
       case Left(diagnostics) => fail(s"getProcessor failed: ${diagnostics.mkString}")
       case Right((diagnostics, tdmlDFDLProcessor)) =>
-        assert(diagnostics.isEmpty)
```

Comment:
@tuxji, this place where I removed diagnostics is just about the getProcessor call and not about parse/unparse diagnostics. Is it safe to keep this one removed? Or maybe I an change it so that it only asserts that there are no error error diagnostics, so that new warnings that get created (like this one) do not break this test?

Reply:
We should fix the relative schemaLocation causing the schema compile-time warning if possible.  If not, let's remove `diagnostics.isEmpty` to make the test more robust.

Reply:
I was hoping to leave all the updates to schemaLocations in a separate PR so that 1) it's easier to look at the real changes if we ever need to come back to it and 2) if we ever add a .git-blame-ignore-revs file then maybe the second commit that updates all the schemaLocations could go in that. I would also like to make sure all tests pass in both commits separately in case we ever need to do a git bisect. So I'd rather we remove or change the assert so this test passes without needing a schemaLocation updatew. I think something like this should work and is maybe reasonable?

```scala
assert(diagnostics.forall(!_.isError))
```
It's maybe the same idea as what the original assert was checking for, but a bit more robust against new warnings that might be added in the future.

Reply:
Yes, sounds good to me